company.el

```diff
@@ -2141,7 +2143,8 @@ With ARG, move by that many elements."
   "Insert the selected candidate."
   (interactive)
   (when (company-manual-begin)
-    (let ((result (nth company-selection company-candidates)))
+    (let* ((selection (or company-selection 0))
```

Comment:
Should we do something with commands like this, too?

Perhaps port even more code from `company-tng`.

Reply:
This command should not run when there's no selection. Checked that.